### ISSUE TITLE
When the first loading response is very slow, a white screen will appear.

### ISSUE THREAD

**Issue by robxyy** [KevinnZou/compose-webview-multiplatform#64]
You can try loading a slow-response URL on `Android`, which happens every time, such as browsing `https://www.google.com` in China.
Ref: https://github.com/google/accompanist/issues/1442#issuecomment-1723305205, there is a way to solve this problem, as follows:
```kt
setLayerType(View.LAYER_TYPE_HARDWARE, null) or setLayerType(View.LAYER_TYPE_SOFTWARE, null)
```

However, there is currently no public API to set layerType, do you have plans for to provide it?

**Comment by KevinnZou** [comment]
@robxyy Thank you for your feedback! I have also identified the problem you mentioned and plan to address it at a later time. I appreciate the solution you provided and will implement it into the project soon.

**Comment by KevinnZou** [comment]
It has been fixed in version 1.7.8
